src/test/server/Server_test.cpp

```diff
@@ -282,13 +282,13 @@ class Server_test : public beast::unit_test::suite
         TestHandler handler;
         auto s = make_Server (handler,
             thread.get_io_service(), journal);
-        std::vector<Port> list;
-        list.resize(1);
-        list.back().port = 0; //let OS select
-        list.back().ip = boost::asio::ip::address::from_string (
-            "127.0.0.1");
-        list.back().protocol.insert("http");
-        auto eps = s->ports (list);
+        std::vector<Port> serverPort {{
+            "testPort",
+            boost::asio::ip::address::from_string ("127.0.0.1"),
+            0,
+            {"http"}
+        }};
```

Comment:
Looks like this construct is failing in visual studio.

Reply:
ok thanks - I'll go back to your original suggestion if I can't make this work.